src/geometry/MeshDrawMgr.cxx

```diff
@@ -66,7 +65,10 @@ MeshDrawMgr::~MeshDrawMgr()
     for (auto& item : lodLists)
     {
         std::for_each(item.begin(), item.end(),
-                      [](auto set){vboElement.vboFree(set);});
+                      [](auto set)
+        {
+            vboElement.vboFree(set);
+        });
```

Comment:
just a note about **astyle**: it has a hard time with some modern C++ constructs, specifically lambda functions and [initializer lists](https://sourceforge.net/p/astyle/bugs/516/). It doesn't seem to be actively maintained. I'm looking for a replacement on my projects.